memdown.js

```diff
@@ -200,6 +205,7 @@ MemDOWN.prototype._iterator = function (options) {
   return new MemIterator(this, options)
 }
 
+MemIterator.prototype._isBuffer =
```

Comment:
Why is this even a prototype method? Wouldn't a simple function be more performant? Are there any implementations overriding it?

Reply:
Historically it was there for the use of `bops`. I also vaguely recall some issue with browserify, but I can't find it. I'll try calling `Buffer.isBuffer` directly; if the browser tests pass we should be good.

Reply:
=> Level/abstract-leveldown#149

Reply:
For reference, there is indeed a browserify issue: https://github.com/Level/abstract-leveldown/issues/172